lua/fzf-lua-frecency/init.lua

```diff
@@ -17,67 +17,82 @@ local M = {}
 
 
 local function get_files_cmd(opts)
+  -- https://github.com/ibhagwan/fzf-lua/blob/e40e2337611fa426b8bcb6989fc310035c6ec4aa/README.md?plain=1#L831-L833
+  local default_fd_opts = [[--color=never --hidden --type f --type l --exclude .git]]
```

Comment:
@ibhagwan I added these default opts to match `FzfLua.files`

Reply:
Good idea